### PR TITLE
trezorlib 0.13.9

### PR DESCRIPTION
**Pull request by matejcik**
builds on #3944
includes #3945

### CODE REVIEW

File: common/tools/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This directory contains mostly tools that can manipulate definitions in [defs/](
 Tools are written with [Click](http://click.pocoo.org/6/), so you always get a help text
 if you use the `--help` option.
 
-All tools require Python 3.6 or higher and a bunch of dependencies, listed in `requirements.txt`.
+All tools require Python 3.8 or higher and a bunch of dependencies, listed in `requirements.txt`.
```
Comment (mmilata) [inline common/tools/README.md:8]: Can we drop 3.6 and 3.7 from shell.nix now?

File: python/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.13.9] (2024-06-19)
+[0.13.9]: https://github.com/trezor/trezor-firmware/compare/python/v0.13.8...python/v0.13.9
+
+### Added
+- trezorctl: Automatically go to bootloader when upgrading firmware  [#2919]
+- Support interaction-less upgrade  [#2919]
+- Added user adjustable brightness setting.  [#3208]
```
Comment (mmilata) [inline python/CHANGELOG.md:14]: Nit: it would be nice to have period at the end consistenly, before it's automated #3973

**Review by mmilata — APPROVED** [review]
LGTM, can't say i fully understand the protobuf field type changes but i suppose any errors would be caught by tests

File: common/tools/README.md (hunk 1 above)
Comment (matejcik) [reply common/tools/README.md:8]: yes

**Pull request merged**